src/main/java/net/sf/jabref/logic/importer/fetcher/CrossRef.java

```diff
@@ -114,16 +111,6 @@ private static boolean checkValidity(BibEntry entry, JSONArray result) {
         }
     }
 
-    private static String removeLaTeX(String text) {
-        String result;
-        // remove braces
-        result = new RemoveBracesFormatter().format(text);
```

Comment:
This also removes the `RemoveBracesFormatter`, which is not part of the computation of latex-free fields. So, if I understand the code correctly, this changes functionality.